src/lib/gql/mutations/vaults.ts

```diff
@@ -31,7 +31,7 @@ export const addVault = (payload: ValueTypes['CreateVaultInput']) =>
                 profile: {
                   address: true,
                   name: true,
-                  users: [{}, { circle_id: true, name: true }],
+                  users: [{}, { circle_id: true }],
```

Comment:
this entire line should be removable now, since without the name, you're not getting any useful data about users